app/helpers/application_helper.rb

```diff
@@ -12,4 +12,8 @@ def input_checked(key, value)
   def facet_has_any_selected_values?(key)
     params.has_key?(key)
   end
+
+  def absolute_url_for(path)
+    URI.join("https://www.gov.uk", path)
```

Comment:
Is there a `Plek` env variable for this? It'd be preferable if this respected host environment, eg dev, preview, prod